MetaMask/metamask-extension
# Fix-Migrate Picker-Network to TS

## Explanation

- FIxes #19124 

## Screenshots/Screencaps

### Before

https://github.com/MetaMask/metamask-extension/assets/79097544/e2c007f3-4a89-40f2-9ff3-4ee6f5faaa69

### After

https://github.com/MetaMask/metamask-extension/assets/79097544/86526c65-6309-481f-9f31-f108f602c966



unit test
![image](https://github.com/MetaMask/metamask-extension/assets/79097544/f6a84215-932c-4872-8a1e-9f750cc7aa21)

## Manual Testing Steps

- Pull this branch
- Check story for `PickerNetwork` component
- Check unit test `yarn jest ui/components/component-library/picker-network`

## Pre-merge author checklist


## Pre-merge reviewer checklist


If further QA is required (e.g. new feature, complex testing steps, large refactor), add the `Extension QA Board` label.

In this case, a QA Engineer approval will be be required.
